tests/RobotFramework/tests/cumulocity/service_monitoring/service_monitoring.robot

```diff
@@ -117,4 +128,19 @@ Check if a service is down
     Should Be Equal    ${SERVICE["type"]}    c8y_Service
    
     Custom Test Teardown
-    
\ No newline at end of file
+
+Check if a service using configured service type
+    [Arguments]    ${service_name}
+    Execute Command    tedge config set service.type thinedge
+    Execute Command    tedge reconnect c8y
```

Comment:
It's a question since I don't remember the implementation details. Here needs `tedge reconnect c8y` because the mosquitto bridge file must be updated?

Reply:
Just restating the `tedge-mapper-c8y` is enough, no need to reconnect here. This is to pick up the new default service type value that is set.

Reply:
fixed